ts/features/bonus/siciliaVola/components/CheckIncomeComponent.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from "react";
+import { useRef } from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { SafeAreaView, ScrollView } from "react-native";
+import { View } from "native-base";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../components/core/typography/H1";
+import { GlobalState } from "../../../../store/reducers/types";
+import {
+  svGenerateVoucherBack,
+  svGenerateVoucherCancel,
+  svGenerateVoucherUnderThresholdIncome
+} from "../store/actions/voucherGeneration";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import { navigateToSvKoCheckIncomeThresholdScreen } from "../navigation/actions";
+import I18n from "../../../../i18n";
+import {
+  RadioButtonList,
+  RadioItem
+} from "../../../../components/core/selection/RadioButtonList";
+
+type OwnProps = {
+  onContinuePress: () => void;
+};
+type Props = OwnProps &
+  ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const getCheckIncomeUnderThresholItems = (): ReadonlyArray<
+  RadioItem<boolean>
+> => [
+  {
+    label: I18n.t("bonus.sv.voucherGeneration.checkIncome.threshold.under"),
```

Comment:
To make sure we have a uniform currency display over the whole app, could be an improvement to use the dedicated function?

```typescript
const threshold = 25000;
const getCheckIncomeUnderThresholdItems = (): ReadonlyArray<
  RadioItem<boolean>
> => [
  {
    label: I18n.t("bonus.sv.voucherGeneration.checkIncome.threshold.under", {
      amount: formatNumberAmount(threshold, true)
    }),
    id: true
  },
  {
    label: I18n.t("bonus.sv.voucherGeneration.checkIncome.threshold.over", {
      amount: formatNumberAmount(threshold, true)
    }),
    id: false
  }
];
```

to do this you need to change the locales too
```yaml
threshold:
    under: "di avere un reddito lordo annuo inferiore o uguale a {{amount}}"
    over: "di avere un reddito lordo annuo superiore a {{amount}}"
```

<img src="https://user-images.githubusercontent.com/822471/128373015-c8845f14-45a5-4176-9fcf-c6ce29518759.png" height="700"/>

Reply:
Yes, for sure! Modified in [9e0a718](https://github.com/pagopa/io-app/pull/3278/commits/9e0a7184518dbfe012a21bfd31b9387560e68394)